### PR TITLE
Project catalogue and about section 

### PR DESCRIPTION
**Pull request by hellohimesh**
1. CSS grid structure created for projects list card
2. project grid finalized, font awesome added in menu and button arrow, 
3. about us section with skills, framework and language

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,49 +11,205 @@
...
+                            <div class="technology-grid-item">
+                                <p> HTML/CSS</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>Ruby on Rails</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>JavaScript</p>
+                            </div>
+                        </div>
```
Comment (andersonlebon) [inline index.html]: - [x] The list of project technologies ("JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>`

File: index.html [hunk 2]
```diff
@@ -11,49 +11,205 @@
...
```
Comment (andersonlebon) [inline index.html]: - [x] The list of project technologies ("JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>`

File: style1.css [hunk 3]
```diff
@@ -13,14 +13,20 @@
...
+  border: none;
+  margin-left: 280px;
+}
+
 .flex-container {
   display: flex;
   align-items: center;
   position: absolute;
+  width: 100%;
 }
```
Comment (andersonlebon) [inline style1.css:31]: - [x] GOOD JOB :clap: your design is looking great. However, you still need to work on the margin of the `project` in my work section. check the screenshots down here :point_down: 

you design     | the original design
------------------ | -----------------------
Screenshot from 2021-11-26 22-13-10 | Screenshot from 2021-11-26 22-14-01

File: index.html [hunk 4]
```diff
@@ -11,49 +11,205 @@
...
+                        <h5>
+                            Project details goes here
+                        </h5>
+                        <div class="technology-grid">
+                            <div class="technology-grid-item">
+                                <p> HTML/CSS</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>Ruby on Rails</p>
+                            </div>
```
Comment (andersonlebon) [inline index.html]: - [x] One of the requirements says that all projects in my work section should be in the `<ul>` tag like this:
```
 <ul class="project-container">
      <li class="project2"></li>
      <li class="project3"></li>
      <li class="project4"></li>
      <li class="project5"></li>
      <li class="project6"></li>
</ul>
``` 
- [x] the work container should also use the `grid` CSS property.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hi @himesh-suthar 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your design design is looking good :+1: :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leolpaz — APPROVED** [review]
Hi @himesh-suthar , 

Your project is complete!
Congratulations! 🎉🎉 

To highlight:

- Congratulations on implementing the changes suggested by the previous reviewer

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**